docs/rules/array-callback-return.md

```diff
@@ -74,7 +74,7 @@ var bar = foo.map(node => node.getAttribute("id"));
 
 ## Options
 
-This rule has an object option:
+This rule has two object options:
```

Comment:
I think this wording is a bit ambiguous, because it could mean that the configuration takes two objects, rather than a single object with two configuration options. Could it be more clear with something like the following?

```suggestion
This rule accepts a configuration object with two options:
```

Reply:
Could we also update the initial paragraph describing the behavior of the rule to also mention this change?

Reply:
Added some more details to the documentation